src/pdn/test/nangate_bsg_black_parrot/dummy_pads.lef

```diff
@@ -172,17 +172,17 @@ MACRO PADCELL_SIG_H
     PORT
       CLASS CORE ;
       LAYER metal4 ;
-        RECT 28.936 139.900 29.096 140.000 ;
+        RECT 23.936 139.900 24.096 140.000 ;
```

Comment:
Why did you change the pin shapes?

Reply:
because the pad is only 25 um wide, so that pin was outside the cell.